.rubocop.yml

```diff
@@ -16,6 +16,9 @@ Lint/Debugger:
   Exclude:
   - lib/cc/cli/console.rb
 
+Lint/AssignmentInCondition:
```

Comment:
I can remove it if we prefer that. I only disabled it because CC reported it and was failing the build.

Do we prefer not to have assignment in condition? If there's no consensus I wouldn't mind making a PR to the style guide.